benchmarks/coremark/Kconfig

```diff
@@ -27,6 +27,32 @@ config COREMARK_STACKSIZE
 	int "CoreMark task stack size"
 	default 4096
 
+config COREMARK_SEED_METHOD
+	int "Coremark Seed Method"
+	default 2
+	---help---
+		Set Coremark Seed Method:
+		0: SEED_ARG
+		1: SEED_FUNC
+		2: SEED_VOLATILE
+
+config COREMARK_MEM_METHOD
+	int "Coremark Memory Method"
+	default 1
+	---help---
+		Set Coremark Memory Method:
+		0: MEM_STATIC
+		1: MEM_MALLOC
+		2: MEM_STACK
```

Comment:
add
```
config COREMARK_MEM_LOCATION
	string
	default "global" if COREMARK_MEM_METHOD == 0
	default "heap" if COREMARK_MEM_METHOD == 1
	default "stack" if COREMARK_MEM_METHOD == 2
```

Reply:
```
apps/benchmarks/coremark/Kconfig:50: error: couldn't parse 'default "GLOBAL" if (CONFIG_COREMARK_MEM_METHOD == 0)': expected symbol
```

it seems that `string` `default` with `equation` does not supported in nuttx Kconfig，only 
```
default "heap" if COREMARK_MEM_METHOD
```
is accepted, should I just keep the original change

```
config COREMARK_MEM_LOCATION
	string "Coremark memory location string"
	default "Defined by the NuttX configuration"
```

or using another type, such like divide these into three Y/N condition which  mapping value of `COREMARK_MEM_METHOD`?


---

```
config COREMARK_MEM_METHOD
	int "Coremark Memory Method"
	default 1
	---help---
		Set Coremark Memory Method:
		0: MEM_STATIC
		1: MEM_MALLOC
		2: MEM_STACK

config COREMARK_MEM_USE_GLOBAL
	bool "Use global memory for Coremark"
	default y if COREMARK_MEM_METHOD == 0
	---help---
	  Use global memory for Coremark

config COREMARK_MEM_USE_HEAP
	bool "Use heap memory for Coremark"
	default y if COREMARK_MEM_METHOD == 1
	---help---
	  Use heap memory for Coremark

config COREMARK_MEM_USE_STACK
	bool "Use stack memory for Coremark"
	default y if COREMARK_MEM_METHOD == 2
	---help---
	  Use stack memory for Coremark

config COREMARK_MEM_LOCATION
	string "Coremark Memory Location"
	default "Defined by the NuttX configuration" if COREMARK_MEM_USE_GLOBAL
	default "heap" if COREMARK_MEM_USE_HEAP
	default "stack" if COREMARK_MEM_USE_STACK
	---help---
	  Set Coremark Memory Location:
	  0: global
	  1: heap
	  2: stack
```

Reply:
Ok, let's define MEM_LOCATION with "if/elif/endif" in Makefile.